ansible/playbooks/tasks/iscsi-client-sbd-prep.yaml

```diff
@@ -44,5 +47,7 @@
 - name: Set LUNs to automatic
   ansible.builtin.command:
     cmd: "iscsiadm -m node -p {{ item.item.stdout | split(':') | first }}:3260 -T {{ srv_iqn }} --op=update --name=node.startup --value=automatic"
+    register: iscsiadm_result
+    changed_when: iscsiadm_result.rc == 0
```

Comment:
This introduce a regression https://openqaworker15.qa.suse.cz/tests/329566#external

```
"msg": "Unsupported parameters for (ansible.legacy.command) module: changed_when, register. 
```